sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### Features Added
 
+- Supported setting default `WebApplicationFirewall` when selected `ApplicationGatewayTier` is `ApplicationGatewayTier.WAF_v2`.
+
 ### Breaking Changes
 
+- Rename class `ExplicitProxySettings` to `ExplicitProxy`.
```

Comment:
nit, "Renamed"

Reply:
Is this the breaking change in their swagger?

Reply:
It's in their commit: https://github.com/Azure/azure-rest-api-specs/commit/8e96f390ad40983947f4608ddc882a20e42444c4

Reply:
Do we expose this class in our convenience code (reason, if yes, we might want to add "rename-model" to avoid break)?

Reply:
It is not exposed in our convenience code.